lib/auth/auth_login_test.go

```diff
@@ -1187,10 +1187,14 @@ func configureForMFA(t *testing.T, srv *TestTLSServer) *configureMFAResp {
 	webDev, err := RegisterTestDevice(ctx, clt, "web-1", proto.DeviceType_DEVICE_TYPE_WEBAUTHN, totpDev)
 	require.NoError(t, err)
 
+	passwordlessDev, err := RegisterTestDevice(ctx, clt, "passwordless-1", proto.DeviceType_DEVICE_TYPE_WEBAUTHN, totpDev, WithPasswordless())
```

Comment:
If this is only for a single test consider moving the device creation there, otherwise every caller of this helper will have to pay the costs of an additional ceremony.